app/src/main/java/org/fossasia/openevent/general/auth/UserDao.kt

```diff
@@ -1,12 +1,17 @@
 package org.fossasia.openevent.general.auth
 
 import android.arch.persistence.room.Insert
+import android.arch.persistence.room.OnConflictStrategy.REPLACE
 import android.arch.persistence.room.Query
+import io.reactivex.Single
 
 interface UserDao {
-    @Insert
+    @Insert(onConflict = REPLACE)
     fun insertUser(user: User)
 
-    @Query("DELETE FROM user_table")
+    @Query("DELETE FROM User")
     fun deleteUser()
+
+    @Query("SELECT * from User")
+    fun getUser(): Single<User>
```

Comment:
There's no use case for selecting all users, and also, all users cannot be selected in a Single<User>